packages/slate-react/src/plugin/dom-helpers.ts

```diff
@@ -0,0 +1,349 @@
+import { Element, Node, Point, Range, Value } from 'slate'
+
```

Comment:
Looks like this file and commands are moved into the main plugin now?

Reply:
Ah yup. I forgot to delete the previously split out ones. Had to combine them to get TypeScript to play nice with extending classes and `this`.